tests/testthat/test-closed_curly_linter.R

```diff
@@ -1,3 +1,10 @@
+closed_curly_message_regex <- rex(
+  paste(
+    "Closing curly-braces should always be on their own line,",
+    "unless they are followed by an else."
+  )
+)
+
```

Comment:
Can we move this into the local test_that scope as to not clutter the global testing env?

Reply:
Done